### DISCUSSION TITLE
you can integrate stuff like this would be muc helpful

### DISCUSSION THREAD

**Discussion by Uddhav07** [rohitsangwan01/uni_control_hub#35] [Ideas]
`
@echo off
adb tcpip 5555
adb connect 192.168.1.20
`

`
@echo off
set IP=192.168.1.20 
for /f "tokens=1 delims=/" %%p in ('nmap %IP% -p 37000-44000 ^| findstr /R "tcp"') do (
	adb connect %IP%:%%p
	exit /b
)
echo No open ADB port found.
`

**Comment by rohitsangwan01** [comment]
Not sure if this is required, as we can connect Wireless ADB on Android 11+ devices with just QrCode scan now
i was planning to implement that actually, with this, we no longer have to connect Android with usb and turn on Wireless ADB
Just scan the qrCode or enter pairing code and done
